### ISSUE TITLE
Make a short ppt for seismology101

### ISSUE THREAD

**Issue by core-man** [seismo-learn/seismology101#58]
I am thinking when we finish the documentation, we may make a ppt so that other can use it to present our documentation.

**Comment by seisman** [comment]
Are you talking about a series of presentations for 3-day workshop or a short 10-page presentation?

**Comment by core-man** [comment]
> Are you talking about a series of presentations for 3-day workshop or a short 10-page presentation?

We may think about the ppt when we finish the documentation. Right now, I mean short 10-page summary.

**Comment by seisman** [comment]
> We may think about the ppt when we finish the documentation. Right now, I mean short 10-page summary.

OK.

Just keep it as a reference, [`pandoc`](https://pandoc.org/) can convert markdown to both PDF (via LaTeX beamer) and PPTX formats. I don't like pptx as it isn't plaintext files, making it difficult to version-control using git.

**Comment by core-man** [comment]
> > We may think about the ppt when we finish the documentation. Right now, I mean short 10-page summary.
> 
> OK.
> 
> Just keep it as a reference, [`pandoc`](https://pandoc.org/) can convert markdown to both PDF (via LaTeX beamer) and PPTX formats. I don't like pptx as it isn't plaintext files, making it difficult to version-control using git.
> 
> https://pandoc.org/MANUAL.html#slide-shows

Great. You are using it to make scientific presentation?

**Comment by seisman** [comment]
> You are using it to make scientific presentation?

No, for scientific presentations, we usually need to add more annotations (arrows, circles), which is impossible/difficult for markdown syntax.

**Comment by seisman** [comment]
Here is a demo https://revealjs.com/?demo
